### PR TITLE
support to wasm 1.0

### PR DESCRIPTION
**Pull request by JoowonYun**
## Summary of changes

<Describe summary of major changes here>

## Report of required housekeeping

- [x] Github issue OR spec proposal link
- [x] Wrote tests
- [ ] Added a relevant changelog entry to CHANGELOG.md

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

**Comment by JoowonYun** [comment]
Pending https://github.com/terra-money/core/pull/652

**Comment by JoowonYun** [comment]
https://github.com/terra-money/core supports 1.0.0. 
Currently, pr supports 1.0.0beta7 and will be updated.

**Review by caramis — APPROVED** [review]
LGTM

File: contracts/terraswap_router/src/contract.rs [hunk 1]
```diff
@@ -225,48 +223,23 @@ pub fn query_config(deps: Deps) -> StdResult<ConfigResponse> {
...
-                    Coin {
-                        denom: offer_denom,
-                        amount: offer_amount,
-                    },
-                    ask_denom,
-                )?;
-
-                offer_amount = res.receive.amount;
+            SwapOperation::NativeSwap { .. } => {
+                return Err(StdError::generic_err("does not support native_swap"));
```
Comment (jhlee-young) [inline contracts/terraswap_router/src/contract.rs:242]: can we update README.md following this change?

File: contracts/terraswap_token/src/contract.rs [hunk 2]
```diff
@@ -28,7 +26,7 @@ pub fn instantiate(
     msg.validate()?;
 
     // create initial accounts
-    let total_supply = create_accounts(&mut deps, &msg.initial_balances)?;
+    let total_supply = create_accounts(&mut deps, msg.initial_balances.as_slice()).unwrap();
```
Comment (jhlee-young) [inline contracts/terraswap_token/src/contract.rs:29]: is `unwrap` better than `?`macro?

File: contracts/terraswap_router/src/contract.rs (hunk 1 above)
Comment (jhlee-young) [reply contracts/terraswap_router/src/contract.rs:242]: how about just removing `NativeSwap`? seems tax querier already has been removed

Comment (JoowonYun) [reply contracts/terraswap_router/src/contract.rs:242]: https://github.com/terraswap/terraswap/pull/44/commits/cfb89576757bb1a0f1f5a4b76061facd1a3380b3

Comment (JoowonYun) [reply contracts/terraswap_router/src/contract.rs]: https://github.com/terraswap/terraswap/pull/44/commits/7e5fd134ec0a38d0af549bba7c36fc47ad6bd38d

File: contracts/terraswap_token/src/contract.rs (hunk 2 above)
Comment (JoowonYun) [reply contracts/terraswap_token/src/contract.rs]: https://github.com/terraswap/terraswap/pull/44/commits/99bf6d48a6fe53f3962b7ba3787f9ab58842c759

File: contracts/terraswap_router/README.md [hunk 3]
```diff
@@ -1,51 +1,58 @@
...
 
-The Router Contract contains the logic to facilitate multi-hop swap operations via native & terraswap.
+The Router Contract contains the logic to facilitate multi-hop swap operations via terraswap.
 
-**On-chain swap & Terraswap is supported.**
+**Only Terraswap is supported.**
 
-Columbus-4 Contract:
-- https://finder.terra.money/columbus-4/address/terra19qx5xe6q9ll4w0890ux7lv2p4mf3csd4qvt3ex
+Pheonixs-4 Contract:
```
Comment (jhlee-young) [inline contracts/terraswap_router/README.md:7]: typo

File: contracts/terraswap_router/src/contract.rs (hunk 1 above)
Comment (jhlee-young) [reply contracts/terraswap_router/src/contract.rs]: needs to update schema as well 🙇‍♀️

File: contracts/terraswap_router/README.md (hunk 3 above)
Comment (JoowonYun) [reply contracts/terraswap_router/README.md]: https://github.com/terraswap/terraswap/pull/44/commits/6c729775cf67596ab9b50b47a29a1be2362b1d23

File: contracts/terraswap_router/src/contract.rs (hunk 1 above)
Comment (JoowonYun) [reply contracts/terraswap_router/src/contract.rs]: https://github.com/terraswap/terraswap/pull/44/commits/6c729775cf67596ab9b50b47a29a1be2362b1d23, https://github.com/terraswap/terraswap/pull/44/commits/43644e09cc40244ee21eb580f43f3d951abcbded

**Review by jhlee-young — APPROVED** [review]

**Pull request merged**